torch/_inductor/ir.py

```diff
@@ -3349,6 +3349,46 @@ def get_template_tiling(self):
         )
 
 
+class AllReduce(ExternKernelAlloc):
+    kernel = "torch.ops.c10d.traceable_allreduce"
+    #1 constant_args show up in codegen, use those if needed
+    #2 allreduce_ can't be called from python, without fixing bindings
```

Comment:
what does this mean? can't you call dist.allreduce directly from python?

Reply:
i switched to dist.allreduce - but dist.allreduce returns a work.

i wanted to initially try calling `c10d.allreduce_`  -- it should be possible, but the bindings are broken.

this isn't blocking me anymore; i think the `dist` route is going to work.